_data/simple-icons.json

```diff
@@ -1650,6 +1650,11 @@
             "hex": "2D72D9",
             "source": "https://www.designernews.co"
         },
+        {
+          "title": "Deutsche Bahn",
+          "hex" : "F01414",
+          "source": "https://www.bahn.de/common/view/static/v8/img/db_em_rgb_100px.svg" 
```

Comment:
Can you fix the indentation please, the linter is currently failing:
![image](https://user-images.githubusercontent.com/12817388/100802360-5327fb80-33f7-11eb-8798-9ccb30586021.png)

Reply:
Thankyou for you review!
I fixed lint